continuousflex/bibtex.py

```diff
@@ -114,5 +113,25 @@
   publisher={Elsevier}
 }
 
+@article{vuillemot2022NMMD,
+title = {NMMD: Efficient Cryo-EM Flexible Fitting Based on Simultaneous Normal Mode and Molecular Dynamics atomic displacements},
+journal = {Journal of Molecular Biology},
+volume = {434},
+number = {7},
+pages = {167483},
+year = {2022},
+issn = {0022-2836},
+doi = {https://doi.org/10.1016/j.jmb.2022.167483},
+url = {https://www.sciencedirect.com/science/article/pii/S0022283622000523},
+author = {Rémi Vuillemot and Osamu Miyashita and Florence Tama and Isabelle Rouiller and Slavica Jonic}
+}
+
```

Comment:
This one does not seem to be cited properly inside scipion (in protocol box Cite, or in methods summary). Could you complete the missing parts?